core-web/libs/data-access/src/lib/dot-contentlet/dot-contentlet.service.ts

```diff
@@ -26,4 +26,19 @@ export class DotContentletService {
             })
             .pipe(take(1), pluck('entity', 'versions', language));
     }
+
+    /**
+     * Get the Contentlet versions by the inode.
+     *
+     * @param string inode
+     * @returns Observable<DotCMSContentlet>
+     * @memberof DotContentletService
+     */
+    getContentletByInode(inode: string): Observable<DotCMSContentlet> {
```

Comment:
Add test for this method.

Reply:
Done, thanks for the feedback! 
https://github.com/dotCMS/core/pull/27663/files#diff-33a1383e93651bf0cbe40ade1692bbf3c38ec6a3c8661da5df0b94b382a32d59R74